demo/micropython-run.ts

```diff
@@ -5,20 +5,37 @@ import { ConsoleLogger, LogLevel } from '../src/utils/logging';
 import { bootromB1 } from './bootrom';
 import { loadUF2, loadMicropythonFlashImage } from './load-flash';
 import fs from 'fs';
+const minimist = require('minimist');
```

Comment:
why require? please change to use `import` as the above

Reply:
updated!